dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/enums/CommandType.java

```diff
@@ -66,11 +69,26 @@ public String getDescp() {
         return descp;
     }
 
-    public static CommandType of(Integer status){
-        for(CommandType cmdType : values()){
-            if(cmdType.getCode() == status){
-                return cmdType;
-            }
+    private static final Map<Integer, CommandType> COMMAND_TYPE_MAP = new HashMap<>();
+
+    static {
+        COMMAND_TYPE_MAP.put(0, START_PROCESS);
+        COMMAND_TYPE_MAP.put(1, START_CURRENT_TASK_PROCESS);
+        COMMAND_TYPE_MAP.put(2, RECOVER_TOLERANCE_FAULT_PROCESS);
+        COMMAND_TYPE_MAP.put(3, RECOVER_SUSPENDED_PROCESS);
+        COMMAND_TYPE_MAP.put(4, START_FAILURE_TASK_PROCESS);
+        COMMAND_TYPE_MAP.put(5, COMPLEMENT_DATA);
+        COMMAND_TYPE_MAP.put(6, SCHEDULER);
+        COMMAND_TYPE_MAP.put(7, REPEAT_RUNNING);
+        COMMAND_TYPE_MAP.put(8, PAUSE);
+        COMMAND_TYPE_MAP.put(9, STOP);
+        COMMAND_TYPE_MAP.put(10, RECOVER_WAITTING_THREAD);
```

Comment:
do you have better idea? I think the map key will be easy to make wrong with the enum self ,  actually , it write twice now, these number  will be easy to wrong

Reply:
I have completed the change, please review

Reply:
> do you have better idea? I think the map key will be easy to make wrong with the enum self , actually , it write twice now, these number will be easy to wrong

I have completed the change, please review